app/Exceptions/Handler.php

```diff
@@ -74,6 +76,34 @@ public function render($request, Exception $exception)
         );
     }
 
+    public function report(Exception $e)
+    {
+        if ($this->shouldntReport($e)) {
```

Comment:
I suggest to rename method `shouldntReport` to `shouldNotReport` to enchance readability.